source/appModules/outlook.py

```diff
@@ -29,33 +29,17 @@
 from NVDAObjects.behaviors import RowWithFakeNavigation, Dialog
 from NVDAObjects.UIA import UIA
 
-oleFlagIconLabels={
-	# Translators: a flag for a Microsoft Outlook message
-	# See https://msdn.microsoft.com/en-us/library/office/aa211991(v=office.11).aspx
-	1:_("purple flag"),
-	# Translators: a flag for a Microsoft Outlook message
-	# See https://msdn.microsoft.com/en-us/library/office/aa211991(v=office.11).aspx
-	2:_("Orange flag"),
-	# Translators: a flag for a Microsoft Outlook message
-	# See https://msdn.microsoft.com/en-us/library/office/aa211991(v=office.11).aspx
-	3:_("Green flag"),
-	# Translators: a flag for a Microsoft Outlook message
-	# See https://msdn.microsoft.com/en-us/library/office/aa211991(v=office.11).aspx
-	4:_("Yellow flag"),
-	# Translators: a flag for a Microsoft Outlook message
-	# See https://msdn.microsoft.com/en-us/library/office/aa211991(v=office.11).aspx
-	5:_("Blue flag"),
-	# Translators: a flag for a Microsoft Outlook message
-	# See https://msdn.microsoft.com/en-us/library/office/aa211991(v=office.11).aspx
-	6:_("Red flag"),
-}
-
-importanceLabels={
-	# Translators: for a high importance email
-	2:_("high importance"),
-	# Translators: For a low importance email
-	0:_("low importance"),
-}
+#: When in a list view, the message classes which should not be announced for an item.
+#: For these, it should be safe to assume that their names consist of only one word.
+silentMessageClasses = [

```

Comment:
So there is no such class for a message/email?

Reply:
Yes, that's the note one. I added a comment that clarifies this.